woocommerce-pdf-invoices-packingslips.php

```diff
@@ -273,11 +272,17 @@ public function need_woocommerce() {
 	/**
 	 * PHP version requirement notice
 	 */
-	
 	public function required_php_version() {
-		$error = __( 'WooCommerce PDF Invoices & Packing Slips requires PHP 5.3 or higher (5.6 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' );
-		$how_to_update = __( 'How to update your PHP version', 'woocommerce-pdf-invoices-packing-slips' );
-		$message = sprintf('<div class="error"><p>%s</p><p><a href="%s">%s</a></p></div>', $error, 'http://docs.wpovernight.com/general/how-to-update-your-php-version/', $how_to_update);
+		$message = sprintf(
```

Comment:
Required PHP version is 5.6, 7.1+ recommended. Maybe make the notice about the addon separate? Or at least conditional for 5.6+